.changeset/stupid-pens-occur.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/core-app-api': patch
+'@backstage/core-components': patch
+'@backstage/plugin-config-schema': patch
+---
+
+Removed the `HomepageTimer` and its related config as it has been replaced by the `HeaderWorldClock` in the Home plugin and was deprecated over a year ago
```

Comment:
```suggestion
**BREAKING**: Removed the `HomepageTimer` and its related config as it has been replaced by the `HeaderWorldClock` in the Home plugin and was deprecated over a year ago
```

Reply:
Updated changeset to include requested changes, thanks :)